terminology/README.md

```diff
@@ -0,0 +1,15 @@
+# Terminology
+
+The language we use has power. Being as concise and descriptive as possible helps to lessen ambiguity. This leads to shared understanding and better outcomes.
+
+## Terms
+
+* Use `blocklist`, and not `blacklist`.
+* Use `allowlist`, and not `whitelist`.
+* Avoid using `master` and `slave`. 
+	* For GitHub, use a term such as `main`, or `publish`, `production`, or `trunk`.
+	* For databases, use `write database` and `read replica`.
+	* For other scenarios, use terms such as `parent` and `child` or `leader` and `follower`.
```

Comment:
I think `parent` and `child` are great examples but I do want to temper them with: be careful what verbs you apply to those. In POSIX we're stuck sending `kill` signals to `children` and it's awkward every time.

Reply:
Do you think it's worth including language about that here? Something like:

> For other scenarios, use terms such as `parent` and `child` or `leader` and `follower`. Be mindful of scenarios where other commands will create language like, sending `kill` signals to `children`.

Reply:
Yes please, and that is perfect!